### PR TITLE
fixes #280

### LINKED ISSUE #280: Dyson Yobb Copyright Info
The copyright info on Dyson Yobb's page hovers mysteriously beside the page, rather than going underneath like it probably should.

### PR DESCRIPTION
**Pull request by mreddekopp**
fixed the footer and aligned it properly

### CODE REVIEW

File: dyobb/index.html [hunk 1]
```diff
@@ -29,9 +29,9 @@
             </p>
             <p>
                 <a href="../">Artist's DB</a>
-            </p><img src="./pics/PortraitTaerha.png" width="100" alt="no alt text">
+            </p><img style="margin-bottom: 0px;" src="./pics/PortraitTaerha.png" width="100" alt="no alt text">
```
Comment (BmanDesignsCanada) [inline dyobb/index.html]: This style should be in css, not in the html.

**Comment by gemohandy** [comment]
I don't fully understand all of the changes, but you removed the mention of a previous editor, and you don't seem to have done anything about the copyright info on any of the other pages.

**Comment by BmanDesignsCanada** [comment]
You should also center the page.

**Comment by gemohandy** [comment]
I think this works? I am not one hundred percent certain. Have you tested it by opening the pages in your browser?

**Comment by mreddekopp** [comment]
bennett plz pull -brett

**Pull request merged**